lib/logo-fetcher.ts

```
@@ -8,244 +8,135 @@
-        if (fetchedBuffer && fetchedBuffer.byteLength > 0) {
-           // TODO: Add basic image validation (is it actually an image?) maybe using sharp?
-           // For now, assume ok if non-empty
-          buffer = fetchedBuffer;
-        } else {
-          fetchError = `Empty response from ${source}`;
-        }
+      // Use a placeholder baseUrl for validation - this is typically used for internal API calls
+      // which won't be made during build time
+      const logoResult = await getLogoFromDataAccess(normalizedDomain, 'http://localhost:3000');
+
+      if (logoResult && logoResult.buffer) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)